pinc/graph_data.inc

```diff
@@ -154,3 +154,73 @@ function tallyboard_deltas($tally_name, $holder_type, $holder_id, $days_back)
         ],
     ];
 }
+
+function new_users($time_interval)
+{
+    $time_interval_options = [
+        'day' => [
+            'format' => '%Y-%b-%d',
+            'title' => _("Number of users registered per day"),
+        ],
+        'week' => [
+            'format' => '%Y-%U',
+            'title' => _("Number of users registered per week"),
+        ],
+        'month' => [
+            'format' => '%Y-%M',
+            'title' => _("Number of users registered per month"),
+        ],
+        'year' => [
+            'format' => '%Y',
+            'title' => _("Number of users registered per year"),
+        ],
+    ];
+
+    $date_format = $time_interval_options[$time_interval]['format'];
+    $title = $time_interval_options[$time_interval]['title'];
+
+    $res = mysqli_query(DPDatabase::get_connection(), "
```

Comment:
I know this is merely a lift of what was there before, but please replace these `mysqli_query()` calls with `DPDatabase::query()` (without the `die()`).

Reply:
Great idea. I was trying to keep diffs minimal, but this makes sense.